### ISSUE TITLE
[prometheus] cAdvisor Stats not included by default

### ISSUE THREAD

**Issue by nrmitchi** [helm/charts#2672]
**Is this a request for help?**:
No

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
Bug Report

**Version of Helm and Kubernetes**:
Helm: v2.6.2
Kubernetes: 1.7.2/1.7.4

**Which chart**:
Prometheus

**What happened**:
Pursuant to https://github.com/kubernetes/kubernetes/issues/48483 (and https://github.com/prometheus/prometheus/pull/2918), cAdvisor stats are not collected by default on newer versions of Kubernetes.

**What you expected to happen**:
The default config map would be up-to-date with the Prometheus documentation

**How to reproduce it** (as minimally and precisely as possible):
 - Install the helm chart
 - Note that the default configuration is missing stats (many of which are used by publicly available Grafana templates)

**Comment by felskrone** [comment]
Im not running prometheus but Heapster and the pod-metrics in grafana stay empty.

The Metrics are available on the kubelets cAdvisor endpoint, but not included in the kubelets metrics.

cAdvisor endpoint
```bash
worker02:/usr/local/bin# curl localhost:4194/metrics
# HELP cadvisor_version_info A metric with a constant '1' value labeled by kernel version, OS version, docker version, cadvisor version & cadvisor revision.
# TYPE cadvisor_version_info gauge
cadvisor_version_info{cadvisorRevision="",cadvisorVersion="",dockerVersion="Unknown",kernelVersion="4.9.0-3-amd64",osVersion="Debian GNU/Linux 9 (stretch)"} 1
# HELP container_cpu_system_seconds_total Cumulative system cpu time consumed in seconds.
# TYPE container_cpu_system_seconds_total counter
container_cpu_system_seconds_total{container_name="",id="/",image="",name="",namespace="",pod_name=""} 1574.56
container_cpu_system_seconds_total{container_name="",id="/init.scope",image="",name="",namespace="",pod_name=""} 0.54
container_cpu_system_seconds_total{container_name="",id="/kube-proxy",image="",name="",namespace="",pod_name=""} 393.93
container_cpu_system_seconds_total{container_name="",id="/kubepods",image="",name="",namespace="",pod_name=""} 259.05
container_cpu_system_seconds_total{container_name="",id="/kubepods/besteffort",image="",name="",namespace="",pod_name=""} 2.27
container_cpu_system_seconds_total{container_name="",id="/kubepods/besteffort/pod4dc3dddd-c5f2-11e7-9179-0022195f92da",image="",name="",namespace="",pod_name=""} 1.63
container_cpu_system_seconds_total{container_name="",id="/kubepods/besteffort/pod4dc3dddd-c5f2-11e7-9179-0022195f92da/6f32b89c25d9c51f5149aefb831fd22df56cbf8a7a0a5d779260977c9c207c43",image="",name="",namespace="",pod_name=""} 1.62
container_cpu_system_seconds_total{container_name="",id="/kubepods/besteffort/pod4dc3dddd-c5f2-11e7-9179-0022195f92da/a4510421700492864b108bc233f0e02e6a31ae837f9cb107956ade9087971175",image="",name="",namespace="",pod_name=""} 0
container_cpu_system_seconds_total{container_name="",id="/kubepods/besteffort/pod4dcbb0b1-c5f2-11e7-9179-0022195f92da",image="",name="",namespace="",pod_name=""} 0.64
container_cpu_system_seconds_total{container_name="",id="/kubepods/besteffort/pod4dcbb0b1-c5f2-11e7-9179-0022195f92da/977111433a45235bbaedfb1eef266c5b4b28924c2edaa7f1b6c71951c5eee686",image="",name="",namespace="",pod_name=""} 0.64
container_cpu_system_seconds_total{container_name="",id="/kubepods/besteffort/pod4dcbb0b1-c5f2-11e7-9179-0022195f92da/cd3af3e753f99569c624b0f3b005fc932479098a7ab36e226d970f1090551d19",image="",name="",namespace="",pod_name=""} 0
container_cpu_system_seconds_total{container_name="",id="/kubepods/burstable",image="",name="",namespace="",pod_name=""} 256.78
container_cpu_system_seconds_total{container_name="",id="/kubepods/burstable/pod43d588ac-c5ef-11e7-9179-0022195f92da",image="",name="",namespace="",pod_name=""} 9.17
container_cpu_system_seconds_total{container_name="",id="/kubepods/burstable/pod43d588ac-c5ef-11e7-9179-0022195f92da/0d14819066e9fb9cc046a1b6b16351df926940ff1e5ee81daa9c2cff0bcf3db0",image="",name="",namespace="",pod_name=""} 1.04
...
```

Kubelet endpoint grepping for '_container' metrics.
```bash
worker02:/usr/local/bin# curl localhost:10255/metrics | grep _cont
# HELP kubelet_containers_per_pod_count The number of containers per pod.
-- -# TYPE kubelet_containers_per_pod_count summary
-:kubelet_containers_per_pod_count{quantile="0.5"} 1
--kubelet_containers_per_pod_count{quantile="0.9"} 1
:-kubelet_containers_per_pod_count{quantile="0.99"} 1
- 5kubelet_containers_per_pod_count_sum 5
95kubelet_containers_per_pod_count_count 3
9k
# HELP kubelet_running_container_count Number of containers currently running
# TYPE kubelet_running_container_count gauge
kubelet_running_container_count 5
kubelet_runtime_operations{operation_type="list_containers"} 758
kubelet_runtime_operations{operation_type="remove_container"} 18
kubelet_runtime_operations_errors{operation_type="remove_container"} 18
kubelet_runtime_operations_latency_microseconds{operation_type="list_containers",quantile="0.5"} 566
kubelet_runtime_operations_latency_microseconds{operation_type="list_containers",quantile="0.9"} 645
kubelet_runtime_operations_latency_microseconds{operation_type="list_containers",quantile="0.99"} 853
kubelet_runtime_operations_latency_microseconds_sum{operation_type="list_containers"} 428796
kubelet_runtime_operations_latency_microseconds_count{operation_type="list_containers"} 758
kubelet_runtime_operations_latency_microseconds{operation_type="remove_container",quantile="0.5"} 5766
kubelet_runtime_operations_latency_microseconds{operation_type="remove_container",quantile="0.9"} 6727
kubelet_runtime_operations_latency_microseconds{operation_type="remove_container",quantile="0.99"} 6894
kubelet_runtime_operations_latency_microseconds_sum{operation_type="remove_container"} 109489
kubelet_runtime_operations_latency_microseconds_count{operation_type="remove_container"} 18
```
I confirmed that for 1.7.6 and 1.7.10.

**Comment by YouriT** [comment]
Isn't #2700 a fix to this?

**Comment by nrmitchi** [comment]
@YouriT Yes, it appears to be. 

This issue can be closed when that gets merged in.

**Comment by nrmitchi** [comment]
Closing because I believe this was addressed in #2700
